javascripts/discourse/initializers/initialize-composer-toggle-hijack.js

```diff
@@ -1,53 +1,240 @@
-import { withPluginApi } from "discourse/lib/plugin-api";
+import { withPluginApi } from 'discourse/lib/plugin-api'
+import discourseComputed from 'discourse-common/utils/decorators'
+
+import {
+  CREATE_SHARED_DRAFT,
+  CREATE_TOPIC,
+  EDIT,
+  PRIVATE_MESSAGE,
+  REPLY,
+} from 'discourse/models/composer'
+
+// Component can get destroyed and lose state
+let _topicSnapshot = null
+let _postSnapshot = null
+let _actionSnapshot = null
 
 export default {
-  name: "toggle-hijack",
+  name: 'toggle-hijack',
 
   initialize() {
     if (settings.hijack_pm_toggle) {
-      withPluginApi("0.8.14", (api) => {
-        api.modifyClass("component:composer-actions", {
+      withPluginApi('0.8.14', api => {
+        api.modifyClass('component:composer-actions', {
           replyAsPrivateMessageSelected(options) {
-            let usernames;
-            let _postSnapshot;
-            let _topicSnapshot;
+            let usernames
+            let _postSnapshot
+            let _topicSnapshot
```

Comment:
Same formatting issue, can we add back the semicolon here`;` as per Discourse convention.